src/icons/GlyphPinCloudFailed.js

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react'
+
+function SvgGlyphPinCloudFailed (props) {
```

Comment:
Can we create a new component that contains the `status === 'Failed' ? GlyphPinCloud : GlyphPinCloudFailed` logic? Maybe in `src/icons/HOCs/GlyphPinCloud.js` ?